pkgs/applications/virtualization/podman-desktop/default.nix

```diff
@@ -3,30 +3,30 @@
 , fetchFromGitHub
 , makeWrapper
 , copyDesktopItems
-, electron
+, electron_33
 , nodejs
-, pnpm
+, pnpm_9
 , makeDesktopItem
 , autoSignDarwinBinariesHook
 , nix-update-script
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "podman-desktop";
-  version = "1.13.2";
+  version = "1.15.0";
 
   passthru.updateScript = nix-update-script { };
 
   src = fetchFromGitHub {
     owner = "containers";
     repo = "podman-desktop";
     rev = "v${finalAttrs.version}";
-    sha256 = "sha256-07lf9jy22JUT+Vc5y9Tu1nkWaXU5RTdu3GibcvQsSs8=";
+    sha256 = "sha256-pewLcukPTXijEP11i2ah8F17N3Rav8UyHrpCu9fUScg=";
   };
 
-  pnpmDeps = pnpm.fetchDeps {
-    inherit (finalAttrs) pname version src;
-    hash = "sha256-LPsNRd1c/cQeyBn3LZKnKeAsZ981sOkLYTnXIZL82LA=";
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit (finalAttrs) pname version src patches;
+    hash = "sha256-EzkAcd84JEBq6eDkh9XCGfCyeo5r2vpZSIvb2hqpsBU=";
```

Comment:
This hash always will be different in nixpkgs-review in x86_64-linux report.

In nixpkgs-review

```
error: hash mismatch in fixed-output derivation '/nix/store/hm84x737h5dnd3g2ky8k5hg6fkmd8ly6-podman-desktop-pnpm-deps.drv':
         specified: sha256-EzkAcd84JEBq6eDkh9XCGfCyeo5r2vpZSIvb2hqpsBU=
            got:    sha256-2wmMuymv8C1ZVeGnWf2R3t9cf1jqiFIw0KJ/m3/cxZs=
error: 1 dependencies of derivation '/nix/store/wgrqfbf8wqkhvk7br091vw2nkfpn94dg-podman-desktop-1.15.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/fiw3n0zkxpwv2zcsc6zc0rya43rs315l-review-shell.drv' failed to build
```

It succeeds in darwin. I have no idea what is this :thinking: